firmware/esp32_imu/esp32_imu.ino

```diff
@@ -588,11 +583,20 @@ void TaskWorker(void *pvParameters)
       quat[2] = q3;
       quat[3] = q0;
       
+      for(int i = 0; i<3; i++)
+      {
+        accel[i] = tmp_accel[i];  
+      }
       if (filter_.getDoBiasEstimation())
       {
         gyro[0] -= filter_.getAngularVelocityBiasX();
```

Comment:
```suggestion
        gyro[0] = tmp_gyro[0] - filter_.getAngularVelocityBiasX();
```
shouldn't this line (and the following two) be changed like this as well ?

Reply:
Yes you are right